draft-ietf-httpbis-priority.md

```diff
@@ -369,70 +380,82 @@ Stream ID:
 Priority Field Value:
 : The priority update value in ASCII text, encoded using Structured Headers.
 
-The HTTP/2 PRIORITY_UPDATE frame MUST NOT be sent prior to opening the
-stream.  If a PRIORITY_UPDATE is received prior to the stream being opened,
-it MAY be treated as a connection error of type PROTOCOL_ERROR.
+The PRIORITY_UPDATE frame MAY be sent before the stream that it references has
+been created. The Stream ID MUST be within the stream limit. If a server
+receives a PRIORITY_UPDATE for a Stream ID that is beyond the stream limits,
+this SHOULD be treated as a connection error of type PROTOCOL_ERROR.
+PRIORITY_UPDATE frames received before the request or response has started
+SHOULD be buffered until the stream is opened and applied immediately after the
+request message has been processed. Holding PRIORITY_UPDATE frames consumes
+extra state on the peer, although the size of the state is bounded by stream
+limits. There is no bound on the number of PRIORITY_UPDATE frames that can be
+sent, so an endpoint SHOULD store only the most recently received frame.
 
-TODO: add more description of how to handle things like receiving
-PRIORITY_UPDATE on wrong stream, a PRIORITY_UPDATE with an invalid ID, etc.
+If a PRIORITY_UPDATE frame is received with a Stream ID of 0x0, the recipient
+MUST respond with a connection error of type PROTOCOL_ERROR.
```

Comment:
Yes. I spotted this when used the HTTP/3 format to describe the full HTTP/2 frame (not just the payload). In that version I called it Prioritized Stream ID to avoid the ambiguation. I'd lost that change by reverting. Thanks for spotting it.